packages/components/src/components/as-histogram-widget/utils/draw.service.ts

```diff
@@ -293,9 +296,31 @@ function _delayFn(_d, i) {
   return i;
 }
 
-export function conditionalFormatter(value) {
-  if (value > 0 && value < 1) {
-    return decimalFormatter(value);
+function getFloatPrecision(value) {
+  const expValue = value.toPrecision()
```

Comment:
```suggestion
  const expValue = value.toString()
```

It looks like without a parameter, `.toString()` is equivalent, right? **https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision**

Reply:
Yes, they are equivalent, it doesn't change anything so it could be replaced if you prefer